src/executables/uber-time.js

```diff
@@ -6,14 +6,15 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 let service = new CommandExecutionService();
 
-program
-  .arguments('<address>')
-  .action(address => {
-    try {
-        service.executeTimeEstimates(address)
-               .then(table => console.log(table));
-    } catch (Error) {
-      console.error('Could not get time estimates');
+program.parse(process.argv);
+
+var address = program.args.toString();
```

Comment:
That would work - but it's important to capture 'tostring' as sometimes someone might put in something like a zip code (which google can geocode) and it'll crap itself.